ci-operator/step-registry/telcov10n/metal-single-node-spoke/gitea/deploy/telcov10n-metal-single-node-spoke-gitea-deploy-commands.sh

```diff
@@ -7,6 +7,8 @@ set -o pipefail
 echo "************ telcov10n Fix user IDs in a container ************"
 [ -e "${HOME}/fix_uid.sh" ] && "${HOME}/fix_uid.sh" || echo "${HOME}/fix_uid.sh was not found" >&2
 
+source ${SHARED_DIR}/spoke-common-functions.sh
```

Comment:
Maybe we should check if this file exists and throw an error if not?

Reply:
Actually, this is the message you already would get whenever the file weren't exist:

`metal-single-node-spoke/site-config/deploy/telcov10n-metal-single-node-spoke-site-config-deploy-commands.sh: line 10: /output/shared_dir/spoke-common-functions-FAKE.sh: No such file or directory`